extensions/blocks/eventbrite/modal-button-preview.js

```diff
@@ -109,6 +120,7 @@ class ButtonEdit extends Component {
 				<RichText
 					placeholder={ placeholder || __( 'Add text…', 'jetpack' ) }
 					value={ text }
+					keepPlaceholderOnFocus={ true }
```

Comment:
Keeping the placeholder text (and fading it out) I think provides a better user experience with buttons. It helps retain the button size when there's no label. Interested in thoughts.

Reply:
I agree, this seems definitely better to me!